storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/queries/DeleteQueryTest.java

```diff
@@ -37,6 +37,13 @@ public void shouldNotAllowNullUriString() {
                 .build();
     }
 
+    @Test(expected = IllegalStateException.class)
```

Comment:
Can you please rewrite tests without using `@Test(expected)`, to check that exception thrown by the `uri()` method and not the `build()` method + check the exception message, otherwise we may brake something in future without noticing that previous behaviour was different!